tests/src/python/test_core_additions.py

```diff
@@ -17,7 +17,7 @@
 
 from qgis.testing import unittest, start_app
 from qgis.core import metaEnumFromValue, metaEnumFromType, QgsTolerance, QgsMapLayer
-import sip
+import PyQt5.sip as sip
```

Comment:
I think this will not work with PyQt < 5.11.
Maybe we'd have to do to something like
import qgis.PyQt.sip as sip
which would work natively for PyQt >= 5.11.
And in core, we integrate the old sip in qgis.PyQt if < 5.11.

Otherwise, there is always the way to hack the modules, but that seems too hacky.